core/src/main/scala/cats/MonoidK.scala

```diff
@@ -38,3 +47,13 @@ import simulacrum.typeclass
       def combine(x: F[A], y: F[A]): F[A] = self.combine(x, y)
     }
 }
+
+trait CompositeMonoidK[F[_],G[_]]
+  extends MonoidK[λ[α => F[G[α]]]] {
+
+  implicit def F: MonoidK[F]
+  implicit def G: MonoidK[G]
```

Comment:
Hmm I guess that seems to be a bit misleading to me. You aren't really composing the instances at all. You are just grabbing a type parameter from the second one. My inclination would be to have `def compose[G[_]]: Semigroup[...]` on `SemigroupK`, `def compose[G[_]]: MonoidK[...]` on `MonoidK`, `def compose[G[_]:Applicative: Applicative[...]` on `Applicative`, and `def compose[G[_]:Applicative]: Alternative[...]` on `Alternative`. To me this seems to be the clearest about what's going on and which constraints are needed.

@non I think you originally added the `SemigroupK[G]` requirement to `compose`. Was that intentional or an oversight? What are your thoughts?